### ISSUE TITLE
HIV

### ISSUE THREAD

**Issue by jpat1546** [biocompute-objects/bco_editor#30]

**Comment by openbox-bio** [comment]
OK, let's discuss.

**Comment by jpat1546** [comment]
Can't attach the json file here, but @openbox-bio sent in an email!
(https://github.com/biocompute-objects/examples/blob/master/HCV1a.json)

**Comment by jpat1546** [comment]
Tested- works! Could you close this issue with a commit @openbox-bio ? Thank you!

**Comment by openbox-bio** [comment]
Hold on. I want the example BCO to appear as a pop up in a new window so
the user can toggle between the "example" window and the "editor" window.
Will let you know once it's done.
-Anjan
=======================================

“Don't be satisfied with stories, how things have gone with others.

Unfold your own myth.”
― Rumi,

========================================

**Comment by jpat1546** [comment]
True. However, now the new window automatically opens when the user selects "Create Object" from the Home Page, and there's no longer the link to "Example BCO"

**Comment by jpat1546** [comment]
As per our conversation, we're modifying the popup to say "Example BCO for reference". 
Thanks @openbox-bio
